### PR TITLE
Minor ion law species update.

### PR DESCRIPTION
**Pull request by Wilchenx**
Adds catpeople to ion laws.

### CODE REVIEW

**Comment by evsey9** [comment]
oh

**Comment by lzimann** [comment]
these click bait titles are annoying

**Comment by The-clown-known-as-Pepe** [comment]
24 hour rule

File: strings/ion_laws.json [hunk 1]
```diff
@@ -827,6 +827,7 @@
     ],
     "ionspecies": [
         "HUMAN BEINGS",
+		"CAT PEOPLE",
```
Comment (lzimann) [inline strings/ion_laws.json]: spaces and not tabs

**Comment by Iamgoofball** [comment]
where's remie and joan complaining that cat people aren't a species?

oh wait, this benefits cat beest so they won't argue it here

**Comment by The-clown-known-as-Pepe** [comment]
ion laws are random as fuck

before this PR catpeople were unaffected by them as """"species""""

**Pull request merged**